manila/share/drivers/netapp/dataontap/cluster_mode/lib_base.py

```diff
@@ -1767,6 +1767,8 @@ def _delete_share(self, share, vserver, vserver_client,
         if force_delete_sec > 0 and duration_sec < force_delete_sec:
             force_delete = True
         if self._share_exists(share_name, vserver_client):
+            if vserver_client.volume_clone_split_status(share_name) != 100:
+                vserver_client.volume_clone_split_stop(share_name)
```

Comment:
can't we simple check the error to be 13171, which means no clone split running. sth like
```
try:
   vserver_client.volume_clone_split_status(...)
except VolumeSplitCloneRunning:
  vserver_client.volume_clone_split_stop(...)
```

Reply:
if clone split not running, the split stop call returns 
       
> <results reason="Unable to stop clone split on volume "xxx" of Vserver "ma_xxx". Reason: Operation is not in progress." status="failed" errno="13001"/>

Reply:
so when clone split is not running, netapp throws 13001 or 13171 ? @chuan137

Reply:
I am not sure above 13171 error. The error is thrown not only when clone split is not running, its thrown also when voluumdoesnot exist. So for all errors we return 100 percentage so that clone stop wont be triggered otherwise actual percentange is returned. Non 100 percentage then will trigger clone split. So chances of getting error 13001 in clone stop are almost zero because that is triggered only if status != 100.

Reply:
check message or better use existing function https://github.com/sapcc/manila/blob/c05497c19996ffe44b67eeeb04ffd28f807fd44c/manila/share/drivers/netapp/dataontap/client/client_cmode.py#L3648-L3650